src/index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Document</title>
+</head>
+<body>
+    Hello, world!
```

Comment:
use 2 whitespaces instead of 4

Reply:
index.html file is changed following your recommendations and uploaded in git again